lib/core-integration/src/Test/Integration/Scenario/API/Byron/Migrations.hs

```diff
@@ -267,7 +267,8 @@ spec = do
             Default
             payloadMigrate >>= flip verify
             [ expectResponseCode @IO HTTP.status202
-            , expectField id ((`shouldBe` 20). length)
+            -- In case of Jormungandr there are only 2 transactions
+            -- , expectField id ((`shouldBe` 20). length)
```

Comment:
Note that we have access to `t` (which stands for 'target') in the context. So you could do something like:

```hs
import Data.Typeable (typeOf)

let numberOfTxs = case show (typeOf (_target ctx)) of 
        s | s == "Proxy * Jormungandr" -> 2 
        s | s == "Proxy * (IO Shelley)" -> 20
        _ -> fail "unknown target backend?" 

[...]
             [ expectResponseCode @IO HTTP.status202 
             , expectField id ((`shouldBe` numberOfTxs) . length)
             ]
```